### PR TITLE
Addlayout

### PR DESCRIPTION
**Pull request by NguyenHuy3003**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +47,6 @@ Configure github action at https://docs.github.com/en/pages/getting-started-with
 3. push pages to github. A build action will be triggered automatically to release the page https://github.com/hippopenny/hippopenny.github.io/actions
 
 
+
+
+
```
Comment (tomcauldron) [inline README.md:52]: can you please add a section with instruction on how to generate pages?

File: _data/alan-wake-ii.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (tomcauldron) [inline _data/alan-wake-ii.json:1]: can we keep all json files related to scores under folder `scores`?

File: _data/animal-crossing-wild-world.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (tomcauldron) [inline _data/animal-crossing-wild-world.json:1]: same as above, this file should be in under folder `scores`

File: _pages/meta-score.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
+---
+title: HippoPenny Score
+layout: scores
+permalink: /meta-score/
+show_excerpts: true
+sort_order: reverse
+sort_by: date
```
Comment (tomcauldron) [inline _pages/meta-score.md:7]: what is the order of posts in the same year, same day? I think you want by hour, minutes (convert datetime to unix timestamp, and sort them)

File: _pages/meta-score.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+         </div>
+        </div>
+     {% assign index = index | plus: 1 %}
+
+    {% endif %}
+  {% endfor %}
+        <p id="indexx" style="display:none;">{{index}}</p>
+
+</div>
+
```
Comment (tomcauldron) [inline _pages/meta-score.md:149]: all these code should be better extracted to .html file and put under `_layouts`. we don't have to do this now though.

File: _pages/meta-score.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+<script>
+
+  var indexx = document.getElementById('indexx').textContent;
+
+  for(let i = 0; i < indexx ; i ++){
+    const score = document.getElementById(`caption1-${i}`).textContent
+    if(score > 90){
+      document.getElementById(`caption1.2-${i}`).innerHTML = "Universal Acclaim"
```
Comment (tomcauldron) [inline _pages/meta-score.md:157]: why don't we add this info like "Universal Acclaim" in the json file and simply read it from there? It's more flexible that way.

File: _pages/meta-score.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
+---
+title: HippoPenny Score
+layout: scores
```
Comment (tomcauldron) [inline _pages/meta-score.md:3]: why don't you put all css styling code in `layout: scores.html`?

File: scores/2023-02-22-alan-wake-ii.md [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+title: HippoPenny Score
+layout: default
+permalink: /meta-score/alan-wake-ii
+show_excerpts: true
+sort_order: reverse
+sort_by: date
+author_profile: false
+---
+
+<style>
```
Comment (tomcauldron) [inline scores/2023-02-22-alan-wake-ii.md:11]: why don't you create a layout file and put all these configs in that file. These .md files should simply set `layout` (line 3).

File: scores/2023-12-26-anno-1800.png
Comment (tomcauldron) [inline scores/2023-12-26-anno-1800.png:1]: if it's possible, i think it's nice to have .png files under folder `assets`

File: README.md [hunk 9]
```diff
@@ -47,3 +47,6 @@ Configure github action at https://docs.github.com/en/pages/getting-started-with
 3. push pages to github. A build action will be triggered automatically to release the page https://github.com/hippopenny/hippopenny.github.io/actions
```
Comment (tomcauldron) [inline README.md:48]: can you please check in scripts to generate posts and cron job script to this repo as well?

File: _data/alan-wake-ii.json (hunk 2 above)
Comment (NguyenHuy3003) [reply _data/alan-wake-ii.json:1]: No, the json file must be saved in _data to be readable

File: README.md (hunk 1 above)
Comment (NguyenHuy3003) [reply README.md:52]: I will add instructions on how to create it manually (previously I planned to write it in the cron section).

File: _data/animal-crossing-wild-world.json (hunk 3 above)
Comment (NguyenHuy3003) [reply _data/animal-crossing-wild-world.json:1]: I tried but it didn't work. The json file can only be read when it is in _data

File: _pages/meta-score.md (hunk 4 above)
Comment (NguyenHuy3003) [reply _pages/meta-score.md:7]: I will change

File: _pages/meta-score.md (hunk 5 above)
Comment (NguyenHuy3003) [reply _pages/meta-score.md:149]: I'll check

File: _pages/meta-score.md (hunk 6 above)
Comment (NguyenHuy3003) [reply _pages/meta-score.md:157]: yep

File: README.md (hunk 9 above)
Comment (NguyenHuy3003) [reply README.md:48]: i will try

**Pull request closed without merging**